test/integration/programs/list/programs-all.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'underscore';
+import moment from 'moment';
+
+context('program all list', function() {
+  specify('display programs list', function() {
+    cy
+      .server()
+      .routePrograms(fx => {
+        fx.data = _.sample(fx.data, 3);
+
+        fx.data[0] = {
+          id: '1',
+          attributes: {
+            name: 'First in List',
+            published: true,
+            updated_at: moment().format(),
+          },
+        };
+
+        fx.data[1].attributes = {
+          name: 'Last in List',
+          published: true,
+          updated_at: moment().subtract(2, 'days').format(),
+        };
+
+        fx.data[2].attributes = {
+          name: 'Secomd in List, Not Published',
```

Comment:
it's the same in both places, so doesn't matter, but "Secomd"

Reply:
Ah, was wondering why the test failed first time around 🙃